string object_name
string goal_name
float64 x
float64 y
string orientation
---
float64 result
